src/archives/dotnet-monitor-archive.proj

```diff
@@ -7,6 +7,11 @@
     <CreateSymbolsArchive>true</CreateSymbolsArchive>
     <ArchiveContentRootPath>$(ArtifactsBinDir)dotnet-monitor\$(Configuration)\$(TargetFramework)\$(PackageRid)\publish\</ArchiveContentRootPath>
   </PropertyGroup>
+  <!-- These items are included in addition to those from publishing the dotnet-monitor project. -->
+  <ItemGroup>
+    <FileToArchive Include="$(RepoRoot)LICENSE.TXT" />
+    <FileToArchive Include="$(ThirdPartyNoticesFilePath)" Condition="Exists('$(ThirdPartyNoticesFilePath)')" />
```

Comment:
I'm actually working on a follow up PR that does this across the board for files that we are explicitly adding (e.g. TPN, license, profiler assemblies, symbols).

Reply:
It's led me to discover that we can't include cross platform files in the archives because they are currently brough in by project publish, which happens during the build legs (which is when the profiler assemblies are also built). I'll have to move that inclusion from publish to archive, but that'll happen in a separate PR since it doesn't current impact either of the outstanding PRs.

Reply:
I'll also want to make the behavior opt-out for local builds.